### DISCUSSION TITLE
Preview intro button

### DISCUSSION THREAD

**Discussion by SK-Fast** [ppy/osu#15216] [Ideas]
Hello.

I have an idea for next upcoming updates. there should be a Intro Preview button in Settings > User Interface > Main Menu > Intro sequence

It's kinda sucks that you need to restart your osu program, So preview intro near Intro sequence setting would be good!
![image](https://user-images.githubusercontent.com/60780448/138229777-c5798d05-406e-4582-92c2-04da9755e6da.png)

**Comment by peppy** [comment]
All settings will eventually have better previews. It's not specific to the intro - a lot of the settings have no way of knowing what they do.
